portfolio/src/main/webapp/index.html

```diff
@@ -7,12 +7,39 @@
     <script src="script.js"></script>
```

Comment:
note: I don't believe karol.png is used anywhere, so it could be removed if you wanted.  I have no concern with leaving it if you want.